src/main/java/data/Account.java

```diff
@@ -111,4 +114,21 @@ public String getAggregationMode() {
   public void setAggregationMode(String aggregationMode) {
     this.aggregationMode = aggregationMode;
   }
+
+  /** Return a String representation of Account fields. */
+  public String toString() {
+    StringBuilder accountBuilder = new StringBuilder();
+    accountBuilder.append("{Legacy_Account_ID:" + getLegacyAccountID());
+    accountBuilder.append(",Next_Gen_Customer_ID:" +  getNextGenAccountID());
+    accountBuilder.append(",Settlement_Attributes:{");
+    accountBuilder.append("Currency_Code:" + getCurrency());
+    accountBuilder.append(",Direction:" + getDirection());
+    accountBuilder.append(",Entity:" + getEntity() + "}");
+    accountBuilder.append(",Settlement_Config:{");
+    accountBuilder.append("Matching_Mode:" + getMatchingMode() + "}");
+    accountBuilder.append(",Account_ID:" + getAccountID());
+    accountBuilder.append(",Aggregation_Mode:" + getAggregationMode() + "}}");
```

Comment:
elsewhere I've been using and seeing snake case for the JSON configs. I don't know if it will make a big difference in the backend, but I think it will make things challenging if these styles are different in the frontend

Reply:
I can change things so be lowercase.